collector/receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -191,6 +188,11 @@ func (t *transaction) AppendHistogram(ref storage.SeriesRef, l labels.Labels, at
 	return 0, nil
 }
 
+func (t *transaction) AppendCTZeroSample(_ storage.SeriesRef, _ labels.Labels, _, _ int64) (storage.SeriesRef, error) {
+	//TODO: implement this func
```

Comment:
We're okay without this I presume?

Reply:
Yup this is something we are working on on the CT working group - its a way of adding explicit resets to the Prometheus TSDB. This doesn't affect the collector at all